### ISSUE TITLE
"name" returns null

### ISSUE THREAD

**Issue by yan-hic** [pdunn/bq_fake_data#1]
Just noticed a "typo" in readme - in case someone uses it as-is:
`     this.val = f.name.findName();`

should read
`     this.name = f.name.findName();`

**Comment by pdunn** [comment]
yup there is a bug in the README thanks for alerting me to it.  The output
variables must match the named fields in the struct.

so where you see
f.val = f.name.findName();
replace that with
f.name = f.name.findName();


I am in process of updating the accompanying medium post, but that example
does work.
https://medium.com/google-cloud/yet-another-way-to-generate-fake-datasets-in-bigquery-93ee87c1008f

CREATE TEMP FUNCTION entity()
RETURNS STRUCT<name String, phone String, ipv6 String, comp_name String>
LANGUAGE js
AS """
  var f = getFaker()
  var t = {};
  t.name = f.name.findName();
  t.phone = f.phone.phoneNumber('1-###-###-###')
  t.ipv6 = f.internet.ipv6();
  t.comp_name = f.company.companyName()
  return t;
"""
OPTIONS (
    library=["gs://<bucket>/bq_fake.js"]
);SELECT entity().*

**Comment by yan-hic** [comment]
Thanks - on that note (updating medium post), what's your take on https://github.com/Marak/faker.js/issues/499#issuecomment-729087078 ? 
I had noticed you used an earlier faker version probably so that it fits within the 1MB limit as packaging a more recent release includes all locales. Hacking the core lib is the only way I found to include some locales only (updating `index.js` or deleting locales).
The `require()` statement in your index.ts only limits what gets loaded in mem, not what goes in the webpack.
Thoughts ?

**Comment by pdunn** [comment]
I'm definitely not a javascript expert.  I thought the original project
uses gulp to separate out the dictionary files.  It would be nice if the
npm libs were also shrunk down.
